apps/src/templates/certificates/StudentsBeyondHoc.story.js

```diff
@@ -1,203 +1,128 @@
-            userType="student"
-            isEnglish={false}
-          />
-        )
-      },
-      {
-        name: 'Dance, signed in, English',
-        description: `Same for young student, older student and teacher`,
-        story: () => (
-          <StudentsBeyondHoc
-            completedTutorialType="dance"
-            userType="student"
-            isEnglish={true}
-          />
-        )
-      },
-      {
-        name: 'Dance, signed in, non-English',
-        description: `Same for young student, older student and teacher`,
-        story: () => (
-          <StudentsBeyondHoc
-            completedTutorialType="dance"
-            userType="student"
-            isEnglish={false}
-          />
-        )
-      },
-      {
-        name: 'Dance, signed out, English',
-        description: `Same for young student, older student and teacher`,
-        story: () => (
-          <StudentsBeyondHoc
-            completedTutorialType="dance"
-            userType="signedOut"
-            isEnglish={true}
-          />
-        )
-      },
-      {
-        name: 'Dance, signed out, non-English',
-        description: `Same for young student, older student and teacher`,
-        story: () => (
-          <StudentsBeyondHoc
-            completedTutorialType="dance"
-            userType="signedOut"
-            isEnglish={false}
-          />
-        )
-      }
-    ]);
+export default {
+  title: 'Congrats/StudentsBeyondHoc',
+  component: StudentsBeyondHoc
+};
+
+const Template = args => (
+  <Provider store={reduxStore()}>
+    <StudentsBeyondHoc {...args} />
+  </Provider>
+);
+
+export const OtherTutorialEnglishTeacher = Template.bind({});
+OtherTutorialEnglishTeacher.args = {
+  completedTutorialType: 'other',
+  isEnglish: true,
+  userType: 'teacher'
+};
+export const OtherTutorialEnglishStudentOver13 = Template.bind({});
+OtherTutorialEnglishStudentOver13.args = {
+  completedTutorialType: 'other',
+  isEnglish: true,
+  userType: 'student',
+  under13: false
+};
+export const OtherTutorialEnglishStudentUnder13 = Template.bind({});
+OtherTutorialEnglishStudentUnder13.args = {
+  ...OtherTutorialEnglishStudentOver13.args,
+  under13: true
+};
+
+export const OtherTutorialEnglishSignedOut = Template.bind({});
+OtherTutorialEnglishSignedOut.args = {
+  completedTutorialType: 'other',
+  userType: 'signedOut',
+  isEnglish: true
+};
+
+export const OtherTutorialNonEnglishSignedOut = Template.bind({});
+OtherTutorialNonEnglishSignedOut.args = {
+  ...OtherTutorialEnglishSignedOut.args,
+  isEnglish: false
+};
+
+export const ApplabSignedInEnglish = Template.bind({});
+ApplabSignedInEnglish.args = {
+  completedTutorialType: 'applab',
+  userType: 'student',
+  isEnglish: true
+};
+
+export const ApplabSignedOutEnglish = Template.bind({});
+ApplabSignedOutEnglish.args = {
+  ...ApplabSignedInEnglish.args,
+  userType: 'signedOut'
+};
+
+export const pre2017MinecraftSignedInEnglishUnder13 = Template.bind({});
```

Comment:
This exported function and the following 2 are lowercase, whereas the others are uppercase. Should we standardize on uppercase exported functions?

Reply:
I just kept the original test name as much as I was able to, but I agree with your point. I am going to change this.